src/main/java/business/services/RequestFormService.java

```diff
@@ -390,6 +390,8 @@ public void transferData(HistoricProcessInstance instance, RequestRepresentation
                 request.setAgreementReached(fetchBooleanVariable("agreement_reached", variables));
                 request.setAgreementNotApplicable(fetchBooleanVariable("agreement_not_applicable", variables));
 
+                request.setRequestType((String)variables.get("request_type"));
```

Comment:
I would prefer to store this value in the `RequestProperties`, as it is not required for the process.